index.html

```diff
@@ -12,74 +12,130 @@
 <body>
   <div class="container">
     <header>
-      <p class="logo">Henry Ik</p>
-      <nav>
-        <img src="images/Union.png" alt="menu" />
-      </nav>
+      <div class=" nav mobile">
+        <p class="logo">Henry Ik</p>
+        <nav>
+          <img src="images/Union.png" alt="menu" />
+        </nav>
+      </div>
+
+      <div class=" nav desktop">
+        <p>Henry Ik</p>
+        <nav>
+          <ul>
+            <a href="#">
+              <li>Portfolio</li>
+            </a>
+            <a href="">
+              <li>About</li>
+            </a>
+            <a href="">
+              <li>Contact</li>
+            </a>
+            <a href=""><img src="images/envelope.png" alt="envelope"></a>
+          </ul>
+        </nav>
+      </div>
+      
     </header>
+
+    
     <main>
-      <h1>Hey There. I'm Henry</h1>
-
-      <section>
-        <h2 class="proffession">I am a Software Developer</h2>
-        <p class="skill">
-          I can help you build a product , feature or website Look through
-          some of my work and experience! If you like what you see and have a
-          project you need coded, don’t hestiate to contact me.
-        </p>
-      </section>
-
-      <section class="social">
+
+      <div class="first-desktop">
+      <div class="intro">
+        <h1>Hey There. I'm Henry</h1>
+        
+          <h2 class="proffession">I am a Software Developer</h2>
+          <p class="skill">
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+        
+      </div>
```

Comment:
It looks fantastic, but you can improve it by following the exact Figma design sizing and spacing values. Please check the difference in the below pictures.

Original
![image](https://user-images.githubusercontent.com/34779928/206297716-5e72b2a5-e9cf-4f4a-b4a2-1ffd4fb9e5c2.png)

Yours
![image](https://user-images.githubusercontent.com/34779928/206297756-7fb21c6d-bc69-4f04-81db-ae76290fb795.png)